lib/GaletteOAuth2/Authorization/UserHelper.php

```diff
@@ -227,8 +245,15 @@ public static function mergeOptions($config, $client_id, array $oauth_scopes)
     // 'email' => 'uuuu@ik.me', 'emailVerified' => NULL, 'phone' => NULL,
     // 'address' => NULL, 'country' => NULL, 'region' => NULL, 'city' => NULL, 'zip' => NULL
 
-    private static function stripAccents($str)
+    public static function stripAccents(string $str): string
     {
-        return \strtr(\utf8_decode($str), \utf8_decode('ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝßàáâãäåæçèéêëìíîïñòóôõöøùúûüýÿĀāĂăĄąĆćĈĉĊċČčĎďĐđĒēĔĕĖėĘęĚěĜĝĞğĠġĢģĤĥĦħĨĩĪīĬĭĮįİıĲĳĴĵĶķĹĺĻļĽľĿŀŁłŃńŅņŇňŉŌōŎŏŐőŒœŔŕŖŗŘřŚśŜŝŞşŠšŢţŤťŦŧŨũŪūŬŭŮůŰűŲųŴŵŶŷŸŹźŻżŽžſƒƠơƯưǍǎǏǐǑǒǓǔǕǖǗǘǙǚǛǜǺǻǼǽǾǿ'), 'AAAAAAAECEEEEIIIIDNOOOOOOUUUUYsaaaaaaaeceeeeiiiinoooooouuuuyyAaAaAaCcCcCcCcDdDdEeEeEeEeEeGgGgGgGgHhHhIiIiIiIiIiIJijJjKkLlLlLlLlllNnNnNnnOoOoOoOEoeRrRrRrSsSsSsSsTtTtTtUuUuUuUuUuUuWwYyYZzZzZzsfOoUuAaIiOoUuUuUuUuUuAaAEaeOo');
+        //FIXME: there is probably a better way to go.
+        //try with something like transliterator_transliterate("Any-Latin; Latin-ASCII; [^a-zA-Z0-9\.\ -_] Remove;", $str);
+        //see https://www.matthecat.com/supprimer-les-accents-d-une-chaine-avec-php.html and https://stackoverflow.com/a/35177682
+        return strtr(
```

Comment:
This change does not work at all, but rogiinal version was not OK neither :/

Just tested with a simple string like `çéè` and the output result was `ace`. :(